Content.Server/GameTicking/Rules/TraitorRuleSystem.cs

```diff
@@ -97,6 +101,14 @@ public bool MakeTraitor(EntityUid traitor, TraitorRuleComponent component, bool
 
         _antag.SendBriefing(traitor, GenerateBriefing(component.Codewords, code, issuer), null, component.GreetSoundNotification);
 
+        // Send codewords to only the traitor client
+        var color = TraitorFallbackColor; // Fall back to a dark red Syndicate color if a prototype is not found
```

Comment:
tbh just use a custom color since the syndie radio channel color doesn't read super well to begin with.